interleave-in-parent.md

```diff
@@ -0,0 +1,174 @@
+---
+title: INTERLEAVE IN PARENT
+summary: Interleaving tables improves query performance by optimizing the key-value structure of closely related table's data.
+toc: false
+toc_not_nested: true
+---
+
+Interleaving tables improves query performance by optimizing the key-value structure of closely related tables, attempting to keep data on the same CockroachDB node if it's likely to be read and written together.
```

Comment:
note: "node" here and elsewhere is correct but not as precise as it could be. I think it's worth introducing ranges earlier, especially since you do end up mentioning them